### PR TITLE
chore: cleanup reading_reminder exp

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes

- experiment won and will be applied only in mobile

## Events

Did you introduce any new tracking events? No

## Experiment

Did you introduce any new experiments? No

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [x] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-447 #done

### Preview domain
https://mi-447-cleanup-reading-reminder.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -142,7 +132,7 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.EditTag);
     }
 
-    if (activeScreen === OnboardingStep.EditTag && showReadingReminder) {
+    if (activeScreen === OnboardingStep.EditTag && isMobile) {
```
Comment (capJavert) [inline packages/webapp/pages/onboarding.tsx:135]: Where was isMobile before or was that decision after experiment won?

**Review by capJavert — APPROVED** [review]

File: packages/webapp/pages/onboarding.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/webapp/pages/onboarding.tsx:135]: It was in growthbook side

Screenshot 2024-07-18 at 11 31 42 AM

Comment (capJavert) [reply packages/webapp/pages/onboarding.tsx:137]: 🚀

**Review by rebelchris — APPROVED** [review]
Nice!
Small change, but slowly making onboarding better :jenkins-triggered:

**Pull request merged**